docs/OpenCart.md

```diff
@@ -0,0 +1,184 @@
+![BTCPay OpenCart: API Keys Permissions](./img/opencart/oc3--06--btcps-generate-api-key-permissions.png)    
+It should look like this:
+![BTCPay OpenCart: API Keys Permissions](./img/opencart/oc3--07--btcps-generate-api-key-permissions-store.png) 
+7. Click on *[Generate API Key]* at the bottom
+8. Copy the generated API Key to your *OpenCart BTCPay settings* form field "BTCPay API Key"
+![BTCPay OpenCart: Copy API Key](./img/opencart/oc3--08--btcps-generate-api-key-result.png) 
+8. Back on BTCPay Server instance go to your store settings and copy the store ID to your *OpenCart BTCPay Settings* form   
+![BTCPay OpenCart: Copy Store ID](./img/opencart/oc3--09--btcps-store-id.png) 
+9. Back on *OpenCart BTCPay settings* form make sure **BTPCay Server URL**, **API Key** and **Store ID** are set and click **[Save]** button (on the top right)    
+![BTCPay OpenCart: Save OpenCart Settings form](./img/opencart/oc3--10--save-settings.png) 
+
+You should get back to the Extensions overview page and see the notification "BTCPay Server Payment details have been successfully updated.". If not make sure your URL, API Key and Store ID are correct.
+![BTCPay OpenCart: Save OpenCart Settings form](./img/opencart/oc3--11--save-settings-success.png) 
+
+On successfully saving the BTCPay extension automatically created a webhook so that OpenCart can get notified when payments settle of fail. To double check it was successful. You can do that by editing the BTCPay extension settings again, if you see the "Webhook Data" field filled out like this:
+![BTCPay OpenCart: Save OpenCart Settings form](./img/opencart/oc3--12--webhook-success.png) 
+
+As you can see on the BTCPay extension settings you can further customize the order statuses depending on the [invoice statuses](https://docs.btcpayserver.org/Invoices/#invoice-statuses) and other common settings. The defaults should be a good starting point but feel free to adjust to your use case. 
+
+## 3. Test the checkout
+
+Everything is ready to go now. Do a small test purchase and make sure the order status gets updated according to the BTCPay invoice status. On BTCPay Server invoice details you can see if the webhook events were fired successfully.
+
+## Troubleshooting
+
+### Enable debug mode
+
+If you have an error during checkout you can enable the debugging mode on the BTCPay extension settings. Menu: Go to "Extensions -> extensions" select "Payments" on the "Choose Extension Type" dropdown and edit BTCPay Server extension.
+
+![BTCPay OpenCart: Enable debug mode](./img/opencart/oc3--20--debug-mode-enable.png) 
+
+You can now find the debug output in the error log in menu "System -> Maintenence -> Error Logs".
+
+![BTCPay OpenCart: Enable debug mode](./img/opencart/oc3--21--error-logs.png) 
+
+*Please make sure to disable it after debugging finished or otherwise it will fill up your error logs.*
```

Comment:
```suggestion
*Please make sure to disable it after debugging is finished; otherwise, it will fill up your error logs.**
```

Reply:
I think now there are two * at the end instead of one or the diff is a bit weird.